server/src/test/java/org/elasticsearch/search/SearchServiceTests.java

```diff
@@ -391,7 +394,7 @@ public void onFailure(Exception e) {
                             );
                             PlainActionFuture<FetchSearchResult> listener = new PlainActionFuture<>();
                             service.executeFetchPhase(req, new SearchShardTask(123L, "", "", "", null, Collections.emptyMap()), listener);
-                            listener.get().decRef();
+                            listener.get();
```

Comment:
Same here, this doesn't need to return anything that has > 0 ref count, just using the future to wait on the action here.